airflow/README.md

```diff
@@ -0,0 +1,49 @@
+# Airflow Orchestration of Dataproc Templates
+
+Content in this folder demonstrates how to run Dataproc Templates from DAGs running in Airflow
+
+1) Structure your Airflow dags folder following this structure:
+```
+dags/
+    |_submit_dataproc_template_serverless.py
+    |_config/
+            |_submit_dataproc_template_serverless.ini
+    |_dependencies/
+            |_dataproc_templates/
+                    |_python/   # Dataproc Templates python folder
+                    |_java/     # Dataproc Templates java folder
+```
+
+2) Update submit_dataproc_template_serverless.ini with the desired template name and its arguments:
+
+```
+[TEMPLATE_ARGS]
+RUNNING_TEMPLATE=<YOUR_DESIRED_TEMPLATED>
+
+[<YOUR_DESIRED_TEMPLATED>]
+template=<YOUR_DESIRED_TEMPLATED>
+your.template.first.argument=<value>
+your.template.second.argument=<value>
+your.template.third.argument=<value>
+your.template.fourth.argument=<value>
+```
+
+3) Run the unit tests:
+```
+pytest airflow/pyspark
+```
+
+4) See the DAG in Airflow with id: **submit_dataproc_template_serverless**
```

Comment:
Is some command or details missing here.
How does `DAG in Airflow` will get deployed ? 
What steps need to be performed for getting to DAG, do someone need to run `_submit_dataproc_template_serverless.py`

Reply:
Improved documentation